Feature/116

afonsobspinto opened the pull request:
Closes #116 

Implemented solution: ... 

How to test this PR: ...

### Sanity checks:
- [ ] The pull request is explicitly linked to the relevant issue(s)
- [ ] The issue is well described: clearly states the problem and the general proposed solution(s)
- [ ] From the issue and the current PR it is explicitly stated how to test the current change
- [ ] The labels in the issue set the scope and the type of issue (bug, feature, etc.)
- [ ] All the automated test checks are passing
- [ ] All the linked issues are included in one milestone
- [ ] All the linked issues are assigned

### Breaking changes (select one):
- [ ] The present changes do not change the preexisting api in any way
- [ ] This PR and the issue are tagged as a `breaking-change`

### Possible deployment updates issues (select one):
- [ ] There is no reason why deployments may break after the current update
- [ ] This PR and the issue are tagged as `alert:deployment`

### Test coverage (select one):
- [ ] Tests for the relevant cases are included in this pr
- [ ] The changes included in this pr are out of the current test coverage scope

### Documentation (select one):
- [ ] The documentation has been updated to match the current changes
- [ ] The changes included in this PR are out of the current documentation scope

### Nice to have (if relevant):
- [ ] Screenshots of the changes
- [ ] Explanatory video/animated gif

zsinnema commented:
@afonsobspinto I've installed feature/116 on salk.dev.metacell.us and there I still see the loading of the files (see my video). can you please check if you have the same?

https://user-images.githubusercontent.com/124044/169467343-2b1b9937-36b9-4b0c-97c6-df0a76a8ffa6.mp4

zsinnema commented:
@afonsobspinto another remark I have is can you explain why you delete inactive populations... I suggest not to remove the inactive population from the local "cache" if someone switches on/off the population to see the effect on the map...

afonsobspinto commented:
As discussed in private, salk.dev.metacell.us was acting as expected according to my implementation.

I agree that:
> the numbers of use cache in comparison to cache invalidation is in favor of use cache

and therefore I added a simple cache to the density map:
https://user-images.githubusercontent.com/19196034/169521832-b2dfdadc-6ed6-4b1a-a2f2-ca8f894a5e28.mp4

I also introduced the prop of invalidCachePopulations to reset the cache when we update the population data through the application (but this last bit wasn't tested because there's no way yet to update the population cells through the application ):
